t/comp/require.t

```diff
@@ -86,7 +86,7 @@ print "# $@\nnot " if $@;
 print "ok ",$i++," - require v5 ignores sub named v5\n";
 
 eval { require 10.0.2; };
-print "# $@\nnot " unless $@ =~ /^\QPerl v10.0.2 required\E/;
+print "# $@\nnot " if $@;
```

Comment:
This change doesn't seem correct. This is meant to check the error from requiring a version number that is too high.

Reply:
You meant this changed the intended meaning of the test? I agree.

I'm not sure if the best solution is to simply delete those tests (there are other tests that do that), or come up with bigger numbers.

Reply:
The easier answer is to massively increase the number to one we'll never hit. Otherwise, calculate a future version and use a string eval to run the require statement.